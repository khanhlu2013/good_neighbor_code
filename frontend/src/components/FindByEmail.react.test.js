import React from 'react';
import FindByEmail from './FindByEmail.react.js';
import TestRenderer from 'react-test-renderer';


test('FindByEmail render correctly',()=>{
  const loginUser = {
    email : 'abc@efg.com',
  }
  const renderer = TestRenderer.create(<FindByEmail loginUser={loginUser}/>);
  const tree = renderer.toJSON();
  expect(tree).toMatchSnapshot();
});

