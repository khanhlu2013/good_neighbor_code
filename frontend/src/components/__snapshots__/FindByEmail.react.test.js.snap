// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FindByEmail render correctly 1`] = `
<div>
  <form
    onSubmit={[Function]}
  >
    <input
      onChange={[Function]}
      placeholder="by email"
      type="text"
      value=""
    />
    <input
      type="submit"
      value="Search friend"
    />
  </form>
</div>
`;
